server/athenian/api/openapi/openapi.yaml

```diff
@@ -3083,11 +3106,25 @@ components:
         stage_timings:
           $ref: '#/components/schemas/StageTimings'
         properties:
-          description: List of the PR properties triggered in the given time window.
+          description: DEPRECATED. List of the PR properties triggered in the given time window.
             There may be extra properties that were not originally requested.
           items:
             $ref: '#/components/schemas/PullRequestProperty'
           type: array
+        events_time_machine:
+          description: List of PR events which happened until `date_to`.
```

Comment:
Is `date_from` ignored?

Reply:
Yes, it is always ignored now.

Reply:
It has been ignored for a few months already, btw.

Reply:
It's not obvious, so it would worth making it explicit.